assets/scss/mixins/_grid.scss

```diff
@@ -6,7 +6,7 @@
 
 @mixin grid {
     display: grid;
-    gap: var(--gutter);
+    gap: 0 var(--gutter);
```

Comment:
IMO we should use `column-gap` https://developer.mozilla.org/fr/docs/Web/CSS/column-gap, therefore we won't have to override row gap if we use it